app/api/config/urls.py

```diff
@@ -1,8 +1,24 @@
 from django.contrib import admin
 from django.urls import include, path
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularSwaggerView,
+    SpectacularRedocView,
+)
 
 urlpatterns = [
     path("api/", include("api.urls")),
     path("api/auth/", include("authn.urls")),
     path("admin/", admin.site.urls),
+    path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
```

Comment:
https://github.com/Health-Informatics-UoN/carrot-mapper/pull/1007#discussion_r2031558222

As before - these should sit in the API project.